e2e/tests/api_tests/sdo/sdo_test.js

```diff
@@ -0,0 +1,183 @@
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(null, config.legalAdvisorUser, 'CREATE_SMALL');
+});
+
+Scenario('1v1 full defence unspecified - judge draws fast track WITH sum of damages', async ({ api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(null, config.judgeUserWithRegionId1, 'CREATE_FAST');
+});
+
+// skipped bc Legal advisor user not yet implemented on environment, so legal advisor tests must be uncommented later
+Scenario.skip('1v1 full defence unspecified - legal advisor draws fast track WITH sum of damages', async ({api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(null, config.legalAdvisorUser, 'CREATE_FAST');
+});
+
+Scenario('1v1 full defence unspecified - judge draws small claims WITHOUT sum of damages', async ({ api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(null, config.judgeUserWithRegionId1, 'CREATE_SMALL_NO_SUM');
+});
+
+// skipped bc Legal advisor user not yet implemented on environment, so legal advisor tests must be uncommented later
+Scenario.skip('1v1 full defence unspecified - legal advisor draws small claims WITHOUT sum of damages', async ({api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(null, config.legalAdvisorUser, 'CREATE_SMALL_NO_SUM');
+});
+
+Scenario('1v1 full defence unspecified - judge draws fast track WITHOUT sum of damages', async ({api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(null, config.judgeUserWithRegionId1, 'CREATE_FAST_NO_SUM');
+});
+
+// skipped bc Legal advisor user not yet implemented on environment, so legal advisor tests must be uncommented later
+Scenario.skip('1v1 full defence unspecified - legal advisor draws fast track WITHOUT sum of damages', async ({api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(null, config.legalAdvisorUser, 'CREATE_FAST_NO_SUM');
+});
+
+Scenario('1v1 full defence unspecified - judge declares SDO unsuitable', async ({api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
+  await api_sdo.createSDO(null, config.judgeUserWithRegionId1, 'UNSUITABLE_FOR_SDO');
+});
+
+// skipped bc Legal advisor user not yet implemented on environment, so legal advisor tests must be uncommented later
+Scenario.skip('1v1 full defence unspecified - legal advisor declares SDO unsuitable', async ({api_sdo}) => {
+  await api_sdo.unspecifiedProcess(config.applicantSolicitorUser, config.defendantSolicitorUser);
```

Comment:
Enable test using legal advisor [tribunal_caseworker_region1@justice.gov.uk](mailto:tribunal_caseworker_region1@justice.gov.uk)/Password12! for central london county court